### ISSUE TITLE
Translucent Flyouts crashes MicaForEveryone

### ISSUE THREAD

**Issue by Andrew-Sor** [ALTaleX531/TranslucentFlyouts#119]
Every time you launch MicaForEveryone with TranslucentFlyouts running earlier, the following error window appears:
![Снимок экрана 2024-03-15 192710](https://github.com/ALTaleX531/TranslucentFlyouts/assets/133566800/b4a64bd5-8746-4419-af19-a322646d4b1b)

Here is the dump file:
[MicaForEveryone.exe-minidump-2024-03-16-14-19-58.dmp](https://github.com/ALTaleX531/TranslucentFlyouts/files/14623362/MicaForEveryone.exe-minidump-2024-03-16-14-19-58.dmp)

If I click "No", TranslucentFlyouts continues to work normally, but MicaForEveryone does not start. If I click "Yes", neither TranslucentFlyouts nor MicaForEveryone work. I have to reset MicaForEveryone to get it to work properly.
How to fix it?

**Comment by ALTaleX531** [comment]
Please check if the latest version has solved your problem.

**Comment by Andrew-Sor** [comment]
Unfortunately no... I suspect that this is due to the fact that I am adding some incompatible process and that is why it crashes. Because when I reset and there are no processes I added in the config, everything is fine... 🤷🏻‍♂️

**Comment by ALTaleX531** [comment]
It's such a strange thing that I can't reproduce the issue in any way. However, you can still add MicaForEveryone to the DisabledList according to the method mentioned in #109. If you find a way to reliably reproduce the issue, please let me know.

**Comment by Andrew-Sor** [comment]
Oh, I found another moment with such an error! You must click "Browse" in the item with the location of the MicaForEveryone configuration file.

**Comment by ALTaleX531** [comment]
Unfortunately, there was no crash and everything was fine.

**Comment by Andrew-Sor** [comment]
Strange... Maybe I didn't install the update correctly?...

**Comment by ALTaleX531** [comment]
@Andrew-Sor It has been confirmed as a MicaForEveryone issue, your configuration file seems to be corrupted.
![image](https://github.com/ALTaleX531/TranslucentFlyouts/assets/105535231/cd3aaa2d-e534-49fe-bbd8-6dbca1cc948f)

**Comment by Andrew-Sor** [comment]
> @Andrew-Sor It has been confirmed as a MicaForEveryone issue, your configuration file seems to be corrupted. ![image](https://private-user-images.githubusercontent.com/105535231/314049944-cd3aaa2d-e534-49fe-bbd8-6dbca1cc948f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA4Njg3OTIsIm5iZiI6MTcxMDg2ODQ5MiwicGF0aCI6Ii8xMDU1MzUyMzEvMzE0MDQ5OTQ0LWNkM2FhYTJkLWU1MzQtNDlmZS1iYmQ4LTZkYmNhMWNjOTQ4Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxOVQxNzE0NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYjBiMzNhZTgyMmE1M2ZkNjlhM2U1MzBmYWZjNDQxMTE5NTk5NmM2YWNhMGUxMDkyNzUxZDdlZGY1NjU4OTBhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.us81Rj0mkyN4u2SzNZi7v82x5HgsWPf1IyBPeCIkilA)

Every time it creates a corrupted configuration file... Thank you, then I will deal with MicaForEveryone. 🙏🏻

**Comment by Andrew-Sor** [comment]
Hmm, I wonder why he wrote about TranslucentFlyouts then... 🤔

**Comment by ALTaleX531** [comment]
If you upgrade to the latest version, you'll see a clearer statement.
> Sorry, this application has thrown an uncaught exception and is about to crash. Note, however, that the crash may not be caused by TranslucentFlyouts, or it may be the application itself.
We have generated a memory dump file in the dumps directory of the folder where TranslucentFlyouts is located named after the current process name as a keyword, which does not contain any of your private information. If you think the crash issue is caused by TF, then we hope you can raise an issue on Github and upload the file, which will help us find and fix the bug quicker.
To prevent the app from crashing constantly, do you want to terminate TranslucentFlyouts.Win32 now?

To avoid problems like #86, TF detects if there are unhandled exceptions, which usually crash the application, and who may come from TF itself or from the application itself. This is a very important safety measure that prevents crashes from happening over and over again without the user knowing what to do.

**Comment by Andrew-Sor** [comment]
It is now clear. Thanks

**Comment by e-t-l** [comment]
@Andrew-Sor I also encounter these error messages when starting TranslucentFlyouts and MicaForEveryone. Can you explain what steps you took to fix your MFE configuration file?

**Comment by Andrew-Sor** [comment]
> @Andrew-Sor I also encounter these error messages when starting TranslucentFlyouts and MicaForEveryone. Can you explain what steps you took to fix your MFE configuration file?

Unfortunately, as I wrote above, I switched to Windows 11 a long time ago and there are no problems there. I really couldn't find any solutions. But I had a little broken explorer.exe due to attempts to install AeroGlass (I probably got a virus or a bad version)... It may be worth trying to look for breakdowns in the system, or you will have to contact the developer himself.
